src/pptrees/adder_tree.py

```diff
@@ -102,6 +107,106 @@ def _hdl_preamble(self):
 
         return (preamble,used_modules)
 
+    def _vhdl_preamble(self):
+        """VHDL preamble for the adder's HDL"""
+
+        preamble = ""
+        used_modules = set()
+
+        # Library imports
+        preamble += "library ieee;\n"
+        preamble += "use ieee.std_logic_1164.all;\n"
+        preamble += "use ieee.numeric_std;\n\n"
+
+        # Entity definition
```

Comment:
Consider a triple quoted string?
```python
        # Entity definition
        preamble += """
entity adder is
  port (
    a,b : in std_logic_vector({0} downto 0);
    cin : in std_logic;
    cout : out std_logic;
    sum : out std_logic_vector({0} downto 0);
  );
""".format(xyz)
```

Reply:
Fair.
I started off concerned about assuming tabs vs using `\t`. I was burned somewhere else in this code by switching computers, recompiling, and having my vim settings replace some tabs to spaces.

So since I made the call to use `\t` explicitly, I decided to also use `\n` explicitly.
For... consistency? But perhaps that is silly.

Reply:
Seeing that your editor has changed tabs to spaces should be easy to see in the git diff?

Reply:
That's fair!